workers/tests/conftest.py

```diff
@@ -30,6 +30,11 @@ def topic_key_minimal() -> str:
     return "minimal"
 
 
+@pytest.fixture(scope="class")
```

Comment:
Since this fixture is only used in one test file let's define it there otherwise this conftest.py file will get very big. And then you only need one fixture, not two fixtures depended on each other.

Reply:
done